firefox-ios/Client/Frontend/Home/JumpBackIn/Data/JumpBackInDataAdaptor.swift

```diff
@@ -89,7 +89,7 @@ actor JumpBackInDataAdaptorImplementation: JumpBackInDataAdaptor, FeatureFlaggab
     // MARK: Jump back in data
 
     private func updateTabsAndAccountData() async {
-        hasSyncAccount = await getHasSyncAccount()
+        hasSyncAccount = getHasSyncAccount()
```

Comment:
Hello! Pretty sure we had a bug with jump back in sync tab whenever the account wasn't asynchronously checked for. I think it was [this bug](https://mozilla-hub.atlassian.net/browse/FXIOS-4753) or [this one](https://mozilla-hub.atlassian.net/browse/FXIOS-4949). Could you have a look and ensure those use case are still working with the changes?

Reply:
I will check those, thanks for the historical context! 🙏 

FYI, I didn't remove awaits... or asynchronous calls... if you look at the code, there was literally no reason I could see to write that code with a completion block. 🤔  It does zero asynchronous work... Very weird!!

Reply:
Both those cases still work fine, here are the demos:

Synced tabs still displayed FXIOS-4753:

https://github.com/user-attachments/assets/f650a961-7883-4623-9882-304617d1ba7b

Synced tab pickup after onboarding FXIOS-4949:

https://github.com/user-attachments/assets/ad890a7e-04f2-497f-a606-3a6dcbd0aeb4

Reply:
I'm not sure why my videos aren't uploading correctly :( I'll DM them to you.

Reply:
I don't know what's up with Github and videos lately, I've seen that issue elsewhere too! I could check the videos on Slack and all looked good ✅